File: leetcode/problems/easy/168.excel-sheet-column-title/kotlin/Solution.kt
// https://leetcode.com/problems/excel-sheet-column-title/
class Solution {
    fun convertToTitle(columnNumber: Int): String {
            val sb = StringBuilder()
            var number = columnNumber
            while(number > 0) {
                number--
                sb.append('A' + number % 26)
                number = number / 26
            }

            return sb.reverse().toString()
    }
}

File: leetcode/problems/easy/168.excel-sheet-column-title/python/solution.py
# https://leetcode.com/problems/excel-sheet-column-title/description/
class Solution:
    def convertToTitle(self, columnNumber: int) -> str:
        chars = []
        while columnNumber > 0:
            columnNumber -= 1
            chars.append(chr(ord('A') + columnNumber % 26))
            columnNumber = columnNumber // 26
        return ''.join(reversed(chars))
